CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and the versioning aims to respect [Semantic Versioning](http://semver.org/spec/
 
 ### Changed
 - Update module files after saving with Protégé 5.6.1(#1628)
+- added process attributes to energy transformation (#1625)
```

Comment:
```suggestion
- energy transformation (#1625)
```
We only write class or property names here. The changes can be tracked in detail via the PR number. This is why a good documentation of the PR description ois required.

Reply:
Please see https://github.com/OpenEnergyPlatform/ontology/wiki/How-to-release-a-new-ontology-version for how to add an "unreleased section" to the changelog.

Reply:
I fixed it.